scene/main/viewport.cpp

```diff
@@ -3316,16 +3316,6 @@ Variant Viewport::gui_get_drag_data() const {
 	return get_section_root_viewport()->gui.drag_data;
 }
 
-PackedStringArray Viewport::get_configuration_warnings() const {
-	ERR_MAIN_THREAD_GUARD_V(PackedStringArray());
-	PackedStringArray warnings = Node::get_configuration_warnings();
-
-	if (size.x <= 1 || size.y <= 1) {
-		warnings.push_back(RTR("The Viewport size must be greater than or equal to 2 pixels on both dimensions to render anything."));
```

Comment:
What happened to this warning?

Reply:
You wrote [a comment](https://github.com/godotengine/godot/pull/90049#discussion_r1769667037) saying it was a dead warning, I didn't see your edit on it until now. I can investigate why it doesn't show up on Window.

Reply:
🤦‍♂️

Reply:
Window size setter was missing `update_configuration_info();` call... but I think it makes more sense to disallow illegal values instead of using a warning for it.

Reply:
Ah right, my EDIT refers to that. SubViewport is capped at 2, but Window is not. Weird tbh.

Reply:
nvm there's more to it than just the missing update call, since Window's "size" field isn't the same as the Viewport's "size" field... same goes for `get_size` not being virtual/override... so it seems like Window needs its own `_get_configuration_info` to check for its own size property :/